modules/drivers/presto-jdbc/src/metabase/driver/presto_jdbc.clj

```diff
@@ -333,10 +335,6 @@
         (log/debug e (trs "Error setting statement fetch direction to FETCH_FORWARD"))))
     stmt))
 
-(defmethod driver/can-connect? :presto-jdbc
```

Comment:
Now I get the error `No method in multimethod 'can-connect?' for dispatch value: :presto-jdbc` when adding a  source database with a new Presto driver.

Reply:
Which commit are you running? Sounds like the driver hasn't been build correctly or is old.

Reply:
The last commit on master.